nimare/meta/mkda.py

```diff
@@ -661,7 +661,7 @@ def correct_fwe_montecarlo(self, result, n_iters=10000, n_cores=-1):
         perm_max_values = perm_results
 
         # Voxel-level FWE
-        vfwe_map = of_values.copy()
+        vfwe_map = of_values.astype(float)
```

Comment:
Ugh, I'm sorry about this. I should have paid closer attention to the actual context here. I think it's a bad idea to use `of_values` here in the first place, since it gives the impression that some OF values will be retained in `vfwe_map` after the for loop below (or even that the `of_values` array is what's being operated on).

Instead, I think we should use something like `np.empty(of_values.shape, dtype=float)`. `float` is the default datatype for `numpy.empty()`, but in this case it might help to be explicit.